src/manifold/src/quickhull.cpp

```diff
@@ -18,56 +18,199 @@
 #include "quickhull.h"
 
 #include <algorithm>
-#include <cassert>
-#include <cmath>
-#include <iostream>
 #include <limits>
 
+namespace manifold {
+
 double defaultEps() { return 0.0000001; }
```

Comment:
Should we change to `kTolerance`? After the double precision PR the epsilon will be much smaller though, we should see how this default epsilon affects precision and runtime.

We can do it in a later PR.